### ISSUE TITLE
website agent stopped generating new events

### ISSUE THREAD

**Issue by shawn8888** [huginn/huginn#2641]
I have created a website agent to generate RSS. It was running OK for a few days, but today it stopped generating new events.
I have tried:
1.	Manually run the agent
2.	Dry run the agent and the result looks good
3.	Delete all old events. 
4.	Change the agent from “on_change” to “All” and run the agent

However the events count stays at 0 and none of above works! Please help me to troubleshoot.

**Comment by shawn8888** [comment]
This is not good. I checked the "Background Jobs" and see a lot of them are in "queued" status. 
So I stopped the docker container ( I run huginn in docker), and I restarted. But now I got an error message like this:
`Docker API has failed. Please visit Docker Log for more information.`

[H.Error.txt](https://github.com/huginn/huginn/files/3941170/H.Error.txt)

**Comment by shawn8888** [comment]
OK problem solved:
1. in Synology docker app, I choose huginn -> Action -> clear
2. After it is done, I start the container huginn again. No ERROR!

I went back to huginn web interface and see the "queued" jobs finished one by one!
I checked the docker log and found this:

`Error,2019/12/09 13:42:50,shawn,Start container huginn failed: {"message":"error while mounting volume '': get d6cec351bc82fac7185f80ce1058452e3344fee4dd9d600a841986ed8186b56d: no such volume"}.
`
So it looks like the container's volume got corrupted somehow and the "clear" action rebuild the volume and fixed the issue!
Hope this can help someone in the same situation.
